geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/package.scala

```diff
@@ -358,6 +358,8 @@ package object filter {
   val MULTICHAR_WILDCARD = "%"
   val SINGLE_CHAR_WILDCARD = "_"
 
+  val WILDCARD_SUFFIX = "\uffff\uffff\uffff"
```

Comment:
@cne1x Does this work for you?  I remember you dropped some unicode knowledge recently...